cmake/HPX_SetupCUDA.cmake

```diff
@@ -22,8 +22,20 @@ if(HPX_WITH_CUDA AND NOT TARGET Cuda::cuda)
     )
   endif()
 
+  # Check CUDA standard
+  if(CMAKE_CUDA_STANDARD)
+    if(CMAKE_CUDA_STANDARD LESS 17)
+      hpx_error(
+        "You've set CMAKE_CUDA_STANDARD to ${CMAKE_CUDA_STANDARD}, which is less than 17 (the minimum required by HPX)"
+      )
+    endif()
+  else()
+    set(CMAKE_CUDA_STANDARD 17)
+  endif()
+
   add_library(Cuda::cuda INTERFACE IMPORTED)
   target_include_directories(Cuda::cuda INTERFACE ${CUDA_INCLUDE_DIRS})
+  target_compile_options(Cuda::cuda INTERFACE cuda_std_${CMAKE_CUDA_STANDARD})
```

Comment:
This should be `target_compiler_features`.

Reply:
Thanks!